app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
   <%= yield :extra_headers %>
+  <%= yield :extra_head_content %>
```

Comment:
Ah, that's a good point. I will rename `extra_headers` to `extra_head_content` and remove the redundant one.